canonical-websites/snapcraft.io
# Rename entrypoint; update production image

- Rename `entrypoint.sh` to `entrypoint` (filename should be language agnostic)
- Use system version of pip3

QA
--

``` bash
./run build
docker build --tag snapcraft --build-arg COMMIT_ID=`git rev-parse HEAD` --build-arg WEBAPP=snapcraft .
docker run -ti -p 8099:80 snapcraft
```

See that it mentions "Booting worker" 3 times. Browse to http://localhost:8099, check it looks okay.
